demo/webpack/prod.js

```diff
@@ -1,12 +1,10 @@
-const devConfig = require('./dev');
+const config = require('./config');
```

Comment:
This is now so small that it almost feels like we could use `--env.prod` and `--env.dev` webpack options instead of having several config files...